pkg/mount/mounter_linux_test.go

```diff
@@ -171,6 +171,9 @@ func validateMount(t *testing.T, mnt string, opts, optional, vfs string) {
 		if mi.Opts != "" {
 			for _, opt := range strings.Split(mi.Opts, ",") {
 				opt = clean(opt)
+				if opt == "relatime" {
+					continue
+				}
 				if !has(wantedOpts, opt) && !has(pOpts, opt) {
```

Comment:
It'll be easier to follow this in the future if we follow the pattern established below with `volunteeredVFS`: add a `volunteeredOpts` map that "knows" that sometimes we get the `relatime` when we don't specifically ask for it, and screen it out here using `!has(volunteeredOpts, opt)` rather than a one-off check.

Reply:
Fixed